### DISCUSSION TITLE
Bug Animate the text

### DISCUSSION THREAD

**Discussion by Eaielectronic** [TurboWarp/extensions#1779] [Show and tell]
Hello, just to let you know that the rainbow effect in animate text doesn't work on all browsers, and in compiled projects it only half works if you move the mouse over it.

**Comment by GarboMuffin** [comment]
can you clarify on which browser it doesn't work?

**Comment by Eaielectronic** [comment]
When the project is in the rainbow package, it doesn't work on any browser and it doesn't work in chrome and firefox mobile and macos sometimes.
